extensions/wikia/UserPreferencesV2/UserPreferencesV2.i18n.php

```diff
@@ -368,6 +367,10 @@
 	'tog-editsectiononrightclick-v2' => 'Abschnitte mit Rechtsklick bearbeiten',
 	'tog-editondblclick-v2' => 'Seiten mit Doppelklick bearbeiten',
 	'tog-hidefollowedpages-v2' => 'Die Liste meiner verfolgten Seiten für jeden ausblenden',
+	'prefs-personal' => 'Meine Informationen',
+	'prefs-wikiemail' => '{{SITENAME}}-Optionen. Sende mir eine E-Mail, wenn …',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "prefs-wikiemail" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)